src/l500/l500-private.cpp

```diff
@@ -147,5 +147,21 @@ namespace librealsense
 
             return rv;
         }
+
+        freefall_option::freefall_option( hw_monitor & hwm )
+            : _hwm( hwm )
+        {
+            // bool_option initializes with def=true, which is what we want
+            assert( is_true() );
+        }
+
+        void freefall_option::set( float value )
+        {
+            bool_option::set( value );
+
+            command cmd{ FALL_DETECT_ENABLE, value != 0.f };
```

Comment:
Float comparison to zero can be tricky - in other places we put
`(fabs(a-b)< std::numeric_limits<float>::epsilon())` template

Reply:
I changed to is_true(), but that compares to the range min. You think we should change the comparison there?